mega-service-agentQnA/validate_agents.sh

```
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Script to validate the functionality of the AgentQnA agent system
+
+# Define working directory if not already set
+if [ -z "$WORKDIR" ]; then
+  WORKDIR=/Users/jduarte/Documents/GenAIBootcamp/free-genai-bootcamp-2025/mega-service-agentQnA
+fi
+
+# Change to the GenAIComps directory
+cd $WORKDIR/GenAIComps
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)